script.js

```diff
@@ -1,4 +1,79 @@
+//include current gamemode
+var currentGameMode = 'insert username to begin';
+//include option for username
+var userName = '';
+// track number of games user wins
+var numUserWin = 0;
+// track number of games user lost
+var numUserLost = 0;
+//track number of games user draws
+var numUserDraw = 0;
```

Comment:
Good use of global variables to keep track of different parameters. Only required to keep track of numUserWin, but implementation of numUserLost and numUserDraw shows understanding of concept of variable value persistency over a program's lifecycle.